sklearn/ensemble/gradient_boosting.py

```diff
@@ -207,13 +271,61 @@ def predict(self, X):
 
 
 class ScaledLogOddsEstimator(LogOddsEstimator):
-    """Log odds ratio scaled by 0.5 -- for exponential loss. """
+    """Log odds ratio scaled by 0.5 -- for exponential loss.
+
+    This estimator takes only the binary target labels of the training set into
+    account. The prediction is independent of the X values in the training set
+    and those passed to the :func:`predict` function.
+
+    It is like :class:`LogOddsEstimator` but takes the log of the *square* of
+    the odds ratio.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.ensemble import gradient_boosting
+    >>> a = np.random.rand(4, 2)
+    >>> y = np.array([1, 1, 0, 1])
+    >>>
```

Comment:
this would confuse the parsers. Doctest's parser expects exactly four characters at the beginning of each line, and you have removed the space at the end. You can simply remove this line.

Reply:
this (and all other empty lines) removed.